plugins/woocommerce-blocks/tests/e2e/tests/mini-cart/mini-cart-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import { expect, test } from '@woocommerce/e2e-playwright-utils';
+import { cli } from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import { openMiniCart } from './utils';
+import { REGULAR_PRICED_PRODUCT_NAME } from '../checkout/constants';
+
+test.describe( 'Shopper → Translations', () => {
+	test.beforeAll( async () => {
+		await cli(
+			`npm run wp-env run tests-cli -- wp language core activate nl_NL`
+		);
+	} );
+
+	test.afterAll( async () => {
+		await cli(
+			`npm run wp-env run tests-cli -- wp language core activate en_US`
+		);
+	} );
+
+	test( 'User can see translation in empty Mini-Cart', async ( {
+		page,
+		frontendUtils,
+	} ) => {
+		await frontendUtils.emptyCart();
+		await frontendUtils.goToShop();
+		await openMiniCart( frontendUtils );
+
+		await expect(
+			page.getByText( 'Je winkelwagen is momenteel leeg!' )
+		).toBeVisible();
+
+		await expect(
+			page.getByRole( 'link', { name: 'Begin met winkelen' } )
+		).toBeVisible();
+	} );
+
+	test( 'User can see translation in filled Mini-Cart', async ( {
+		page,
+		frontendUtils,
+	} ) => {
+		await frontendUtils.goToShop();
+		await page.getByLabel( 'Toevoegen aan winkelwagen: “Beanie“' ).click();
+		await openMiniCart( frontendUtils );
+
+		await expect(
+			page.getByRole( 'heading', { name: 'Je winkelwagen (1 artikel)' } )
+		).toBeVisible();
+
+		await expect(
+			page.getByRole( 'link', { name: 'Mijn winkelmand bekijken' } )
+		).toBeVisible();
+
+		await expect(
+			page.getByRole( 'link', { name: 'Naar afrekenen' } )
+		).toBeVisible();
+	} );
+} );
+
+// Skipping tax tests until we sorted out a bug that hides the totals price on hover
```

Comment:
Could you add a reference to the ticket here?

Reply:
Ah, good catch, @mikejolley. @Aljullu actually resolved this issue with #43550. I'll update this test accordingly.